engine/src/main/java/org/camunda/bpm/engine/impl/cmd/HistoryCleanupCmd.java

```diff
@@ -60,13 +58,14 @@ public HistoryCleanupCmd(boolean immediatelyDue) {
   }
 
   @Override
-  public Job execute(CommandContext commandContext) {
-    if (!isHistoryCleanupEnabled(commandContext)) {
+  public Job execute(CommandContext context) {
+
+    if (!context.isHistoryCleanupEnabled()) {
```

Comment:
🔧 To simplify the code a bit and remove `#isHistoryCleanupEnabled` from `CommandContext`, we can do the following:
```java
    ProcessEngineConfigurationImpl processEngineConfiguration = context.getProcessEngineConfiguration()
    if (!processEngineConfiguration.isHistoryCleanupEnabled()) { ... }
    AuthorizationManager authorizationManager = context.getAuthorizationManager();
    authorizationManager.checkCamundaAdmin();
```

Reply:
Done